test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -2674,8 +2674,8 @@ public async Task SetStatusOrchestration(bool extendedSessions, string storagePr
                 await client.WaitForCustomStatusAsync(TimeSpan.FromSeconds(10), this.output, statusValue);
 
                 // Test clearing an existing custom status
-                await client.RaiseEventAsync("UpdateStatus", null, this.output);
-                await client.WaitForCustomStatusAsync(TimeSpan.FromSeconds(10), this.output, JValue.CreateNull());
+                await client.RaiseEventAsync("UpdateStatus", "", this.output);
+                await client.WaitForCustomStatusAsync(TimeSpan.FromSeconds(30), this.output, JValue.CreateNull());

```

Comment:
Currently a hack. I found that `RaiseEventAsync` was failing when the `eventData` argument was `null`. It sent a message reading something like "value cannot null". Did something change in our implementation of `RaiseEventAsync`?

Reply:
Yeah, there were changes with how we handled `null` objects serializing in DurableTask.Core. I'm looking into the best way to fix it now. https://github.com/Azure/durabletask/pull/465/files.

Reply:
I am going to fix that before I merge https://github.com/Azure/azure-functions-durable-extension/pull/1577.